evm_loader/cli/src/logs.rs

```diff
@@ -32,26 +33,32 @@ const LOG_MODULES: [&str; 11] = [
 
 
 pub fn init(context: LogContext) -> Result<(), log::SetLoggerError> {
-    let mut dispatch: Dispatch =
-        fern::Dispatch::new()
-            .level(log::LevelFilter::Error);
+
+    let mut dispatch: Dispatch = fern::Dispatch::new().level(log::LevelFilter::Error);
 
     for module_name in LOG_MODULES {
         dispatch = dispatch.level_for(module_name, log::LevelFilter::Trace);
     }
 
     dispatch
         .format(move |out, message, record| {
+            let line: String = record.line().map_or("NA".to_string(), |v| v.to_string());
+
+            let _path = record.file().unwrap_or("Undefined");
+            let file_path = path::Path::new(_path).file_name()
+                                                  .map_or("Undefined", |v| v.to_str().unwrap());
```

Comment:
This may be written just a bit better:
```
            let file_path: &str =
                record.file()
                    .and_then(|filepath| path::Path::new(filepath).file_name() )
                    .and_then(|filename| filename.to_str() )
                    .unwrap_or("Undefined");

Reply:
Look much more better Thank you very much!